src/main/java/hudson/plugins/s3/Entry.java

```diff
@@ -113,11 +113,11 @@ public Entry(String bucket, String sourceFile, String excludedFile, String stora
 
     @Override
     public Descriptor<Entry> getDescriptor() {
-        return DESCRIPOR;
+        return DESCRIPTOR;
     }
 
     @Extension
-    public static final DescriptorImpl DESCRIPOR = new DescriptorImpl();
+    public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
```

Comment:
I *do* hope that no one was using the `DESCRIPOR` field directly before..

Reply:
I don't think that's an expected use case, but I'll revert.

Reply:
No it's fine. Normally, scripts shouldn't be using it.